runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -113,4 +113,25 @@ The terminal cursor can be highlighted via |hl-TermCursor| and
 |hl-TermCursorNC|.
 
 ==============================================================================
+5. Status Variables				    *terminal-emulator-status*
+
+Terminal buffers maintain some information about the terminal in buffer-local
+variables:
+
+- *b:term_title* The settable title of the terminal, typically displayed in
+  the window title or tab title of a graphical terminal emulator. Programs
+  running in the terminal can set this title via an escape sequence.
+- *b:terminal_job_id* The nvim job ID of the job running in the terminal. See
+  |job-control| for more information.
+- *b:terminal_job_pid* The PID of the top-level process running in the
+  terminal.
+
+These variables will have a value by the time the TermOpen autocmd runs, and
+will continue to have a value for the lifetime of the terminal buffer, making
+them suitable for use in 'statusline'. For example, to show the terminal title
+as the status line:
+>
+	:autocmd TermOpen * setlocal statusline=%{b:term_title}
```

Comment:
This was indented with a tab, actually; quite happy to switch to spaces, though.